### PR TITLE
ALCF fixes: tiff_to_zarr.py dynamic pixelsize, init_tiff_to_zarr_globus_flow.py, fix for issue #27

### PR DESCRIPTION
**Pull request by davramov**
- Updated `tiff_to_zarr.py` to read pixelsize from the raw h5 file and convert to micrometer for the multiscales step.
- Updated `init_tiff_to_zarr_globus_flow.py` to account for the new change (passing in the h5 path to the flow) as well as automatically updating the Prefect Secret Block on the server with the latest `flow_id` and `func_id` once the script runs. Side note: I removed the time import from the function definition because I was getting a cryptic "`time.run()` is not defined error" despite only calling `time.time()`.
- Updated instructions for running `check_globus_compute.py` in the docstring.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208451498137898

### CODE REVIEW

File: scripts/polaris/tiff_to_zarr.py [hunk 1]
```diff
@@ -58,8 +71,8 @@ def main():
     backend = detect_cli_io_backend(file_paths)
     image = cli_input_to_ngff_image(backend, file_paths)
     # The scale and axis units are the same as the one printed in the reconstruction script
-    image.scale = {'z': 0.65, 'y': 0.65, 'x': 0.65}
-    image.axes_units = {'z': 'micrometer', 'y': 'micrometer', 'x': 'micrometer'}
+    image.scale = read_pixelsize_from_hdf5(args.raw_directory)
+    image.axes_units = {'x': 'micrometer', 'y': 'micrometer', 'z': 'micrometer'}
```
Comment (dylanmcreynolds) [inline scripts/polaris/tiff_to_zarr.py:75]: I think you can get units from the h5 file as well.

**Review by dylanmcreynolds — CHANGES_REQUESTED** [review]

File: scripts/init_tiff_to_zarr_globus_flow.py [hunk 2]
```diff
@@ -31,9 +31,6 @@ def conversion_wrapper(rundir, h5_file_name, folder_path):
     """
     import os
     import subprocess
-    import time
```
Comment (dylanmcreynolds) [inline scripts/init_tiff_to_zarr_globus_flow.py:34]: Was the issue that references time.run() at the import or the use of `time.time()`

That's super duper odd. I would be interested to see `print(time.time)` if it's no occuring during the import.

Comment (davramov) [reply scripts/init_tiff_to_zarr_globus_flow.py:34]: I couldn't recreate the issue locally but only when the Globus Flow called the function. There error listed specifically the line `start = time.time()`. What's odd to me is A) it was working before and B) I am calling time.time() and nowhere do I call time.run().

File: scripts/polaris/tiff_to_zarr.py (hunk 1 above)
Comment (davramov) [reply scripts/polaris/tiff_to_zarr.py:75]: The pixel_size attribute seems to only store the numerical value but not the units. The name of the camera model is also listed in the h5 file (PCO_Edge), but not the units. The units are defined in a comment in one of Dula's Jupyter notebooks (`notebooks/microct/backend/ALS_recon_functions.py`)

Here are the attributes stored in `/measurement/instrument/detector/` in the h5 file:
```{'binning_x': array([1.]),
'binning_y': array([1.]),
'dark_field_value': array([100.]),
'delay_time': array([0.]),
'dimension_x': array([2560.]),
'dimension_y': array([2160.]),
'exposure_time': array([0.15, 0.15, 0.15, ..., 0.15, 0.15, 0.15]),
'model': b'PCO_Edge',
'pixel_size': array([0.00065]),
'temperature': array([25.])}```

**Comment by dylanmcreynolds** [comment]
Fixes #27

**Review by dylanmcreynolds — APPROVED** [review]

**Pull request merged**